shopping-cart/app/com/commercetools/sunrise/framework/checkout/payment/DefaultCheckoutPaymentControllerAction.java

```diff
@@ -102,4 +103,9 @@ private PaymentMethodInfo findPaymentMethod(final List<PaymentMethodInfo> paymen
                 .findAny()
                 .orElseThrow(() -> new RuntimeException("No valid payment method found")); // Should not happen after validation
     }
+
+    private boolean hasReferencesExpanded(final PaymentInfo paymentInfo) {
+        return paymentInfo.getPayments().stream()
+                .allMatch(reference -> reference.getObj() != null);
```

Comment:
Here it would be nice if our `Reference` class would provide a `isExpanded` method, this would then allow us to use a method handle here: `.allMatch(reference::isExpanded)`. 

WDYT?